server/helpers/articlePayload.js

```diff
@@ -0,0 +1,43 @@
+import sequelize from 'sequelize';
+import models from '@models';
+
+const { User, Rating } = models;
+
+
+const articlePayload = {
```

Comment:
get the updated version of this payload from @EMacco

Reply:
Syncing with @EMacco . Thanks for your kind observation.